be/src/exprs/block_bloom_filter.hpp

```diff
@@ -40,11 +44,14 @@ class BlockBloomFilter {
     explicit BlockBloomFilter();
     ~BlockBloomFilter();
 
+    BlockBloomFilter(const BlockBloomFilter&) = delete;
+    void operator=(const BlockBloomFilter&) = delete;
```

Comment:
```suggestion
    BlockBloomFilter& operator=(const BlockBloomFilter&) = delete;
```

According to [copy_assignment](https://en.cppreference.com/w/cpp/language/copy_assignment).